exe/kubernetes-run

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'kubernetes-deploy'
+require 'kubernetes-deploy/runner_task'
+require 'optparse'
+
+namespace = "default"
+command = nil
+ARGV.options do |opts|
+  opts.on("--namespace=NAMESPACE") { |n| namespace = n }
+  opts.on("--command") { |c| command = c }
+  opts.parse!
+end
+
+runner = KubernetesDeploy::RunnerTask.new(
+  namespace: namespace
+)
+
+runner.run(
+  task_template: ARGV[0],
+  command: command,
+  args: ARGV[1..-1]
```

Comment:
I think they can be passed wholesale to the container as the `args`. Any potential env vars could use a `--env` flag to demarcate them specifically.